package.json

```diff
@@ -11,6 +11,8 @@
     "examples": "npm run build && http-server",
     "test": "nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/*.spec.ts test/**/*.spec.ts",
     "test:fullstory": "nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/fullstory.spec.ts",
+    "test:browser": "PLAYWRIGHT_BROWSERS_PATH=0 DLO_RUN_BROWSER_TESTS=1 nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/*.spec.ts -- --grep 'CEDDL to FullStory'",
```

Comment:
Couple notes here:
- `PLAYWRIGHT_BROWSERS_PATH=0` installs browser binaries in `node_modules/playwright-core/.local-browsers`. I took this route to isolate the impact of DLO Playwright browser tests on the broader dev environment. Interested in feedback on this approach.
- I'm expecting `--grep 'CEDDL to FullStory'` to evolve in subsequent PRs to hybridize other ruleset tests for execution in node or browser environments.

Reply:
On Bullet Point 1, 
I don't see any issues here, and makes sense to me. You are essentially changing the default cache folder that Playwright would normally download to (ie: `~/Library/Caches/ms-playwright` for Mac Machines).

 This might make sense to do for the `eco-test-party` as well as I imagine having multiple repositories writing to the default cache location could cause some version issues or something like that.